### DISCUSSION TITLE
Is there a react component to render the markdown in a js page?

### DISCUSSION THREAD

**Discussion by LintaoAmons** [facebook/docusaurus#9802] [Q&A] [answered]
So I want to render a markdown the same way it rendered in the docs or blogs, but in a js Page.

**Comment by Josh-Cena** [comment, marked as answer]
No. Docusaurus only compiles Markdown once at build time. If your Markdown is statically available at build time, you can import it:

```js
import Document from "./document.mdx";

<Document />
```

If you want to dynamically parse and render arbitrary Markdown, you have to bring your own Markdown parsing library.

**Reply by LintaoAmons** [reply]
Okie... Got, it~
